single_tech_samples/fabric/fabric_ci_cd/README.md

```diff
@@ -0,0 +1,140 @@
+# Introduction 
+
+This repo contains the code for establishing a CI/CD process around Fabric workspaces. The code is intended to be used as a jumpstart for a new project on Microsoft Fabric. Currently, There are many limitations, but the goal is to expand the capabilities of the code over time.
+
+## Deployment Process
+
+### Understanding "bootstrap" script
+
+### Infrastructure deployment steps
+
+Here are the steps to use the bootstrap script:
+
+1. Change the directory to the `scripts` folder:
+
+```bash
+cd scripts
+```
+
+1. Rename the [env.sample](./.env.sample) file to `.env` and fill in the necessary environment variables. Here is the detailed explanation of the environment variables:
```

Comment:
We also need to add  a second step with the same for params.json-template and refer that the admin user needs to be in the tenant you are testing on and that the capacity name is a new name you want to create in Azure.